app/components/landing-page/MeetTheDevTeam/index.tsx

```diff
@@ -6,7 +6,7 @@ import Image from 'next/image'
 
 const MeetTheDevTeam = () => {
```

Comment:
Is it meet the dev team, or meet the KA team?

Reply:
<img src="https://user-images.githubusercontent.com/69960893/142578897-5ab0d5f3-6b28-487d-b781-fdd3e6d060a9.png">

Meet The Dev Team is for this button. 

<img src="https://user-images.githubusercontent.com/69960893/142579087-fb60c692-295a-49c7-b40c-42cb66d5e653.png">

MeetKaTeam is for the ka team page.

Reply:
> Meet The Dev Team is for this button.  

How about change it to KA instead of dev?